lib/dbutils.js

```diff
@@ -13,6 +13,21 @@ dbu.conversions = {
         read: JSON.parse,
         type: 'blob'
     },
+    string: {
+        write: function(value) {
```

Comment:
What's the use case here? If we define the field to be a string-like type in the table, I don't think this would be needed.